src/chrome/content/rules/OpenX.org.xml

```diff
@@ -1,60 +1,37 @@
 <!--
-	For other OpenX coverage, see OpenX.xml.
-
-
-	CDN buckets:
-
-		- akamai.openx.org
-
-			- a1051.w7.akamai.net
-			- blog.openx.org
-			- support.openx.org
-
-
-	Nonfunctional hosts in *openx.org:
-
-		- forum *
-
-	* Shows blog.openx.org; self-signed, CN: *.openx.com
-
-
-	Problematic hosts in *openx.org:
-
-		- ^ **		(works, self-signed)
-		- adserver *
-		- blog *
-		- edit-blog **
-		- support	(shows blog.openx.org, akamai)
-		- www		(Redirect destination mismatched)
-
-	* Works, akamai
-	** Works; self-signed, CN: *.openx.com
-
-
-	Fully covered hosts in *openx.org:
-
-		- adserver
-
+	Other OpenX.com related rulesets, see
+		+ OpenX.xml
+
+	Non-functional hosts
+		Timeout was reached:
+		- codex.openx.org
+		- confluence.openx.org
+		- dc.openx.org
```

Comment:
I think we don't need to list a host if it is not functional in both http and https.